docs/installation/oracle.md

```diff
@@ -15,21 +15,25 @@ Support subscription to install Docker on Oracle Linux.
 
 This page instructs you to install using Docker-managed release packages and
 installation mechanisms. Using these packages ensures you get the latest release
-of Docker. If you wish to install using Oracle-managed packages, consult your
-[Oracle Linux documentation](https://oracle.com/linux).
+of Docker. If you wish to install using Oracle-managed packages, consult the
+[Oracle Linux documentation](http://docs.oracle.com/en/operating-systems/?tab=2).
```

Comment:
Yeah, https is fine too. My bookmark was made on the http site so I just cut and paste it. :)